simple/imm_data.c

```diff
@@ -73,7 +73,7 @@ static int alloc_cm_res(void)
 	cm_attr.wait_obj = FI_WAIT_FD;
 	ret = fi_eq_open(fab, &cm_attr, &cmeq, NULL);
 	if (ret)
-		printf("fi_eq_open cm %s\n", fi_strerror(-ret));
+		printf("fi_eq_open() cm %s\n", fi_strerror(-ret));
```

Comment:
Agreed. I was going for a minimum distance / quick change to clear up some confusion with some internal users/testers.

Sent from my phone. No type good.

On Jan 30, 2015, at 1:10 PM, Sean Hefty <notifications@github.com<mailto:notifications@github.com>> wrote:


In simple/imm_data.c<https://github.com/ofiwg/fabtests/pull/137#discussion_r23859888>:

> @@ -73,7 +73,7 @@ static int alloc_cm_res(void)
>       cm_attr.wait_obj = FI_WAIT_FD;
>       ret = fi_eq_open(fab, &cm_attr, &cmeq, NULL);
>       if (ret)
> -             printf("fi_eq_open cm %s\n", fi_strerror(-ret));
> +             printf("fi_eq_open() cm %s\n", fi_strerror(-ret));


That would work.

—
Reply to this email directly or view it on GitHub<https://github.com/ofiwg/fabtests/pull/137/files#r23859888>.